_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,144 @@
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/alias-creation.png" alt="create an alias"/>{: .img-fluid }
+
+### Simplify the flow for the reindex operation
+
+You can select the source and destination indexes, aliases, or data streams from the dropdown menu as seen in the following image. Moreover, we provide an ad-hoc destination index creation flow that allows you to import settings and mappings directly from the source.
+
+![Image: Reindex page]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/reindex.png){:.img-fluid }
+
+### Shrink index operation
+
+The **Shrink index** operation is used to reduce the number of primary shards in an existing index and create a new index.
+
+To shrink an index, go to the **Indices** page, select an index, and choose the **Shrink** action in the **Actions** menu. This will take you to the shrink index page as seen in the following image.
+
+![Image: Shrink-action]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-action.jpg){:.img-fluid }
+Only one index can be shrunk at once, and data stream indexes are not supported for a shrink operation. If multiple indexes are selected or a data stream backing an index is selected in the **Indices** page, the **Shrink** option is disabled.
+{: .note }
+
+Before shrinking an index, ensure that the index is not in one of the following states:
+
+* The index's health status is red.
+* The index has only one primary shard.
+
+If the index is in any of the above states, error messages will be displayed on the **Shrink index** page, and the **Shrink** button will be disabled as seen in the following images.
+
+![Image: Shrink-red]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-red.jpg){:.img-fluid }
+
+![Image: Shrink-one-shard]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-one-shard.jpg){:.img-fluid }
+
+The **Shrink index* operation requires certain prerequisites to be met. If the source index does not meet the specified conditions, error messages will be displayed on the **Shrink index** page.
+
+The source index must block write operations, which means that you need to set the `index.blocks.write` setting in the source index to `true`. If the source index is not set to block write operations, you can choose the **Block write operations** button to set the `index.blocks.write` setting to `true` as seen in the following image.
+
+![Image: Shrink-block-write]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-block-write.jpg){:.img-fluid }
+
+The source index must also be open. If the source index is closed, the shrink operation will fail. You can choose the **Open** button to open the index as seen in the following image. This may take additional time to complete, and the index will be in the red health status while opening.
```

Comment:
```suggestion
The source index must also be open. If the source index is closed, the shrink operation will fail. You can choose the **Open** button to open the index, as shown in the following image. This may take additional time to complete, and the index will be in the red health status while opening.
```